src/main/java/org/opencds/cqf/cds/discovery/DiscoveryElementR4.java

```diff
@@ -36,8 +36,12 @@ public JsonObject getAsJson() {
 
             if (prefetchUrlList != null && !prefetchUrlList.isEmpty()) {
                 JsonObject prefetchContent = new JsonObject();
-                prefetchContent.addProperty("item1", "Patient?_id={{context.patientId}}");
-                int itemNo = 1;
+                int itemNo = 0;
+                if (!prefetchUrlList.stream().anyMatch(p -> p.startsWith("Patient"))) {
+                    prefetchContent.addProperty("item1", "Patient?_id={{context.patientId}}");
```

Comment:
rather than startsWith here, can we use equals on the explicit patterns? "Patient/{{context.patientId}}" and "Patient?_id={{context.patientId}}"? I'm concerned that there could be more specific requests that this would override and would result in really difficult to find bugs.